src/Bundle/CoreBundle/Tests/Field/TextAreaTypeTest.php

```diff
@@ -15,6 +15,31 @@ public function testContentTypeFieldTypeWithEmptySettings()
         $this->assertCount(0, $errors);
     }
 
+    public function testAllowedRowsSetting()
+    {
+        $field = $this->createContentTypeField('textarea');
+        $field->setSettings(
+          new FieldableFieldSettings(
+            [
+              'rows' => 'abc',
+            ]
+          )
+        );
+        $errors = $this->container->get('validator')->validate($field);
+        $this->assertCount(1, $errors);
+        $this->assertEquals('settings.rows', $errors->get(0)->getPropertyPath());
+        $this->assertEquals('validation.nointeger_value', $errors->get(0)->getMessage());
+
+        $field->setSettings(
+          new FieldableFieldSettings(
+            [
+              'rows' => 20,
+            ]
+          )
+        );
+        $this->assertCount(0, $this->container->get('validator')->validate($field));
```

Comment:
You should add a test, to check if setting the row number actually changes the form options attribute array. see https://github.com/unite-cms/unite-cms/blob/master/src/Bundle/CoreBundle/Tests/Field/ReferenceFieldTypeTest.php#L115